solutions/kubernetes/helm/chartrepo/nginx-repo/Readme.md

```diff
@@ -29,83 +28,118 @@ curl -SL https://github.com/docker/compose/releases/download/v2.16.0/docker-comp
 chmod +x /usr/local/bin/docker-compose
 ```
 
-#### **Volumes**
+#### Helm Installation
 
-For this setup two volumes are required:
+Please check whether helm is exist in your system. You can verify that using below command.
 
-- Configuration Volume:
-
-  - This volume is to override the default configuration of NGINX.
-  - The default location if set to **/nginx/conf**.
-  - You can override that in docker-compose.yaml file.
-  - Copy the default.conf file to that location
+```
+helm version
+```
 
-- Repository Volume:
-  - This volume to add the actual Helm charts and serve them through NGINX.
-  - The default location is **/nginx/repos**.
-  - You can override that in docker-compose.yaml file.
+Please use the below steps for installing helm if not already installed. You can also take reference from https://helm.sh/docs/intro/install/#from-script.
 
-<a name="packaging-chart" />
+```
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+```
 
-## Package the Chart
+<a name="setup-nginx-server" />
 
-For packaging the chart we will take example of Helm chart for DevOps Dashboard project.
-**Note:** You can also use your own chart
+## Setup NGINX server
 
-1. Clone the “dashboard” branch from “cortx-re” repository.
+1. Clone the “dashboard” branch from “cortx-re” repository
 
 ```
 git clone https://github.com/Seagate/cortx-re.git -b dashboard
 ```
 
-2. Go to the “cortx-re/solutions/dashboard/charts” directory.
+2. Goto **nginx-repo** directory
 
 ```
-cd cortx-re/solutions/dashboard/charts
+cd cortx-re/solutions/kubernetes/helm/chartrepo/nginx-repo
 ```
 
-3. Package dashboard-project helm chart
+3. Create directories for volume mapping with NGINX server
 
 ```
-helm package dashboard-project/
+mkdir -p /nginx/conf
+mkdir -p /nginx/repos
 ```
 
-4. Package elastic-project helm chart
+4. Copy default.conf file to **/nginx/conf** directory
 
 ```
-helm package elastic-project/
+cp default.conf /nginx/conf
 ```
 
-Step 3 and 4 will generate **.tgz** file and these files we will put in the repository.
+5. Start the NGINX server
+   By default below configuration applied:
+   Port: **30080**
+   Volumes: **/nginx/conf/**, **/nginx/repos/**
+
+   If you want to modify the configuration then please update the **docker-compose.yaml** file.
+
+```
+docker-compose up -d
+```
+
+Open the http://\<host_url>:\<port> in the browser and check the directory structure.
 
 <a name="create-helm-repository" />
 
 ## Create Helm Repository
 
-Create the directory structure in below format in the **Repository Volume** mentioned above.
+1. Create **chartrepo** directory under **/nginx/repos/**
 
 ```
-chartrepo
-├─── index.yaml
-├─── chart_category_name
-  └─── chart_package.tgz
+mkdir -p /nginx/repos/chartrepo
+```
+
+2. Create the directory named as the **dashboard-charts** in **chartrepo**
+
+```
+mkdir -p /nginx/repos/chartrepo/dashboard-charts
```

Comment:
can't we merge creating both directories mentioned in step-1 and step-2 in one command?

Reply:
Yes we can, I will merge

Reply:
Done